lib/authentication/auth_default.js

```diff
@@ -10,19 +10,39 @@
  * @returns {Object}
  * @constructor
  */
-function AuthDefault(password) {
+function AuthDefault(connectionConfig) {
+  const password = connectionConfig.password;
+  const mfaToken = connectionConfig.mfaToken;
+  const passcode = connectionConfig.getPasscode();
```

Comment:
The mfatoken is configured on the connection level (Because we need to check that the cleintMFARequest option is on and the token exists). That is why it does not have the getter for mfaToken in the connection config for the mfaToken field. I have no clue why the connector has used the field for the password. Might be a security reason? If you want to make a getter for those two fields, I will have a look. Please share your idea.

Reply:
It was only question about why some fields have getters but another don't have. It isn't incorrect.